src/msg/DispatchQueue.cc

```diff
@@ -96,6 +96,7 @@ void DispatchQueue::enqueue(const ref_t<Message>& m, int priority, uint64_t id)
 
 void DispatchQueue::local_delivery(const ref_t<Message>& m, int priority)
 {
+  m->set_throttle_stamp(ceph_clock_now());
```

Comment:
shall we `set_recv_complete_stamp()` as well?

Reply:
@tchaikov I think this is entirely reasonable and consistent however it raises the question of order. @jdurgin suggested the current order although looking at it more deeply this is inconsistent so I'm wondering if we want the order here to be consistent with the order in ProtocolV2.cc and ProtocolV1.cc files or do we want to change their order to bring them in line with this file. I would suggest the change with the least impact is to copy the order from ProtocolV2.cc into DispatchQueue::local_delivery() and that's what I'll do for now however I'm more to make any further changes in terms of ordering anyone considers appropriate but I think we should attempt to be consistent unless there's a very good reason not to be.